src/components/QrcodeStream.vue

```diff
@@ -133,22 +133,63 @@ export default {
   },
 
   methods: {
+    async setCameraById(deviceId) {
```

Comment:
So how would the user set the deviceId? By calling this function on the component? Hmm, I think best practice would be to do this through props. And then this would require an update to the documentation.